core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionFileSystemOutput.java

```diff
@@ -109,7 +109,7 @@ public TransmissionFileSystemOutput(String folderPath, String maxTransmissionSto
                                                                         DEFAULT_CAPACITY_MEGABYTES,
                                                                         MAX_TRANSMISSION_STORAGE_CAPACITY_NAME,
                                                                         maxTransmissionStorageCapacity);
-        capacityInBytes = capacityEnforcer.getCurrentValue() * 1024 * 1024;
+        capacityInBytes = ((long)capacityEnforcer.getCurrentValue()) * 1024L * 1024L;
```

Comment:
similar to above, this cast is not needed, or is it for clarity?
```suggestion
        capacityInBytes = capacityEnforcer.getCurrentValue() * 1024L * 1024L;
```

Reply:
I couldn't recall if the cast was needed or not. Since the other operands are `long`, this cast is redundant.

For reference: [Java Language Spec, Section 5.6.2](https://docs.oracle.com/javase/specs/jls/se7/html/jls-5.html#jls-5.6.2)